packages/usa-in-page-navigation/src/index.js

```diff
@@ -14,6 +14,7 @@ const IN_PAGE_NAV_TITLE_HEADING_LEVEL = "h4";
 const IN_PAGE_NAV_SCROLL_OFFSET = 0;
 const IN_PAGE_NAV_ROOT_MARGIN = "0px 0px 0px 0px";
 const IN_PAGE_NAV_THRESHOLD = "1";
+const IN_PAGE_NAV_MINIMUM_HEADING_COUNT = 3;
```

Comment:
Hm now I'm thinking of a scenario where even if there are only 2 headers maybe the content is longer form in which the navigation might be helpful to flag that there's more content below?  Although if that's a minor edge case the user can just utilize the new data attribute.

Reply:
The issue itself says _don't_ display if there are one or two headings.

Reply:
The issue does say that. However, I think we need to reconsider setting 3 as the hard-coded default, since it runs a much higher risk of becoming a breaking change. 

If we set the hard-coded default value to 3, a lot of current implementations of the component will disappear from the page unless the user takes action. I would rather users choose to turn off the components according to their site's content strategy, rather than being forced to turn it on. 

IMO, simply adding this attribute will resolve the issue by opening the door for the user to choose that they want to have a minimum of three headings. I don't think we need to hard code it.

Reply:
### Decision

After internal discussions we decided a minimum of two headings was a good starting point.

- Two headings fits a real user need (the USWDS website's [z-index page](https://designsystem.digital.gov/design-tokens/z-index/)) and is mentioned in the original issue.
- Single headings are usually at the top, so wouldn't make sense to have a minimum of zero or one.
- This is a dynamically generated enhancement that **doesn't** impact critical content.
- It would save users from additional custom code code to hide the empty menu.

Reply:
This update was made in [fdbd228](https://github.com/uswds/uswds/pull/6205/commits/fdbd2281ac74231198d9b1521a0ce2f90427c250)